docs/source/model_sharing.rst

```diff
@@ -78,7 +78,9 @@ Once you are logged in with your model hub credentials, you can start building y
 
     transformers-cli repo create your-model-name
 
-This creates a repo on the model hub, which can be cloned.
+This creates a repo on the model hub, which can be cloned. If you want to create a repo under a specific organization,
+you might find the `upload_a_new_repo.py script
+<https://github.com/huggingface/efficient_scripts/blob/main/upload_a_new_repo.py>`__ useful.
```

Comment:
I'm not sure this should be here. Wouldn't it be more logical to write:

If you want to create a repo under a specific organization:

    transformers-cli repo create your-model-name --organization your-org-name

Here's a script that exposes the API calls that we do in case you want to do it programmatically:

(your link)

Reply:
oh yeah, you're completely right (should have waited before merging :-/) Didn't check that the `--organization` flag exists. I'll fix that tomorrow :-)